src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -65,80 +75,132 @@
+              @change="onJurisdictionChange($event)"
+            />
+          </v-col>
+        </v-row>
+
+        <!-- Name in Home Jurisdiction -->
+        <v-row
+          class="mt-6"
+          no-gutters
         >
-          <p id="home-jurisdiction">
-            <label>Home Jurisdiction: </label>
-            <span>{{ homeJurisdiction || '[Unknown]' }}</span>
-          </p>
-
-          <p id="business-name">
-            <label>Business Name: </label>
-            <span>{{ businessName }}</span>
-          </p>
-
-          <p id="business-number">
-            <label>Business Number: </label>
-            <span>{{ businessNumber || '[Unknown]' }}</span>
-          </p>
-
-          <p id="incorporation-date">
-            <label>Incorporation Date: </label>
-            <span>{{ incorporationDate || '[Unknown]' }}</span>
-          </p>
-        </v-col>
-      </v-row>
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <label>Name in Home Jurisdiction</label>
+          </v-col>
+
+          <v-col
+            cols="12"
+            sm="9"
+          >
+            <v-text-field
+              v-model="business.homeLegalName"
+              class="name-home-jurisdiction"
+              filled
+              hide-details="auto"
+              label="Name in Home Jurisdiction"
+              :rules="getShowErrors ? businessNameRules : []"
+            />
+          </v-col>
+        </v-row>
 
-      <template v-if="isBusinessActive">
+        <!-- Date of Incorporation, Continuation or Amalgamation in Home Jurisdiction -->
         <v-row
-          class="mt-4"
+          class="mt-6"
           no-gutters
         >
           <v-col
             cols="12"
             sm="3"
           >
-            <label>Type of Business in Home Jurisdiction</label>
+            <label>Date of Incorporation, Continuation or Amalgamation in Home Jurisdiction</label>
           </v-col>
 
           <v-col
             cols="12"
             sm="9"
           >
-            <v-radio-group
-              v-model="business.isUlc"
-              class="mt-0 pt-0"
-              row
-            >
-              <v-radio
-                label="Unlimited Liability Company"
-                :value="true"
-              />
-              <v-radio
-                label="Other"
-                :value="false"
-              />
-            </v-radio-group>
+            <DatePickerShared
+              id="incorporation-date"
+              ref="incorporationDateRef"
+              title="Date of Incorporation, Continuation or Amalgamation in Home Jurisdiction"
+              :nudgeRight="40"
+              :nudgeTop="85"
+              :persistentHint="true"
+              :initialValue="business.homeIncorporationDate"
+              :inputRules="getShowErrors ? incorporationDateRules: []"
+              :maxDate="getCurrentDate"
+              @emitDateSync="$set(business, 'homeIncorporationDate', $event)"
```

Comment:
Let me call your attention to this `$set` call, which adds the prop "homeIncorporationDate" to the "business" object reactively (since business was initialized to `{}`).

Without this, the new date was not recognized by the watcher function and the valid event was not emitted to the parent component.

Reply:
I see now, this is setting `business.homeIncorporationDate` to whatever date is being picked reactively. Sweet.

Reply:
Simply assigning the value correctly updated the business object but watchers (and computeds, etc) did not react to it, so this was needed. It's because the business object was initialized without properties.